src/booleans.js

```diff
@@ -1,64 +1,92 @@
+
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  return a && b;
 };
 
 function either(a, b) {
-  // your code here
+  return a || b; 
 };
 
 function none(a, b) {
-  // your code here
+  if (a === false && b === false) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function one(a, b) {
-  // your code here
+  if (a === true && b === false) {
+    return true;
+  } else if (a === false && b === true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function truthiness(a) {
-  // your code here
-};
+  return Boolean(a);
+}; 
```

Comment:
Whitespace at the end of the line should be removed
```suggestion
};
```